airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -348,6 +348,11 @@ class OAuthAuthenticator(BaseModel):
         examples=['%Y-%m-%d %H:%M:%S.%f+00:00'],
         title='Token Expiry Date Format',
     )
+    token_expiry_is_time_of_expiration: Optional[bool] = Field(
+        None,
+        description='Indicates that the Token Expiry returns the date until which the token will be valid, not the amount of time it will be valid.',
```

Comment:
```suggestion
        False,
        description='Indicates that the Token Expiry returns the date until which the token will be valid, not the amount of time it will be valid.',
```
IMO, it should be clear from the schema what is the default value, not hidden inside of the internal class. This rule should be applied to all declarative configuration

Reply:
Agree, updated code with default `False` value for token_expiry_is_time_of_expiration